sdk/sqlmanagement/Azure.ResourceManager.Sql/src/Custom/VirtualClusterResource.cs

```diff
@@ -31,6 +31,7 @@ public partial class VirtualClusterResource
         /// Operation Id: VirtualClusters_UpdateDnsServers
         /// </summary>
         /// <param name="cancellationToken"> The cancellation token to use. </param>
+        [ForwardsClientCalls]
         public virtual async Task<Response<ManagedInstanceUpdateDnsServersOperationData>> UpdateDnsServersAsync(CancellationToken cancellationToken = default) =>
             await (await UpdateDnsServersAsync(WaitUntil.Started, cancellationToken).ConfigureAwait(false)).WaitForCompletionAsync(cancellationToken).ConfigureAwait(false);
```

Comment:
For line 36, why not use `WaitUntil.Completed` directly?

Reply:
If we use WaitUntil.Completed, we need to construct the Response<T> by ourselves like this:
```csharp
public virtual async Task<Response<ManagedInstanceUpdateDnsServersOperationData>> UpdateDnsServersAsync(CancellationToken cancellationToken = default)
{
    var operation = await UpdateDnsServersAsync(WaitUntil.Completed, cancellationToken).ConfigureAwait(false);
    return Response.FromValue(operatoin.Value, operatoin.GetRawResponse());
}
```
Personally, I prefer the first implementation so I use `WaitUntil.Started` here.